tasks/install_shell.sh

```diff
@@ -137,7 +137,7 @@ if [ -n "$PT_mac_source" ]; then
   mac_source=$PT_mac_source
 else
   if [ "$nightly" = true ]; then
-    mac_source='http://nightlies.puppet.com/downloads'
+    mac_source='https://artifactory.delivery.puppetlabs.net:443/artifactory/internal_nightly__local/downloads'
```

Comment:
This is fine for testing, but it will need to use the new public API for downloading artifacts before merging.

Reply:
We talked about this and there's a separate ticket to add support for the new endpoint, so I'll resolve this.